### ISSUE TITLE
Unwanted trailing blank lines in @clean nodes

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#1048]
at.putBody now regularizing trailing whitespace more simply.  This is made possible because Leo no longer generates `@nonl` sentinels.

The same code fixes both #1048 and #1037.

**Comment by edreamleo** [comment]
Fixed in the "output" branch. It will be merged only after more testing.

**Comment by edreamleo** [comment]
The "output" branch has been fully merged into "devel".
